bodhi/server/consumers/automatic_updates.py

```diff
@@ -140,19 +141,33 @@ def __call__(self, message: fedora_messaging.api.Message) -> None:
 
             log.debug(f"Creating new update for {bnvr}.")
             changelog = build.get_changelog(lastupdate=True)
+            closing_bugs = []
             if changelog:
+                log.debug("Adding changelog to update notes.")
                 notes = f"""Automatic update for {bnvr}.
 
 ##### **Changelog**
 
 ```
 {changelog}
 ```"""
+
+                for b in re.finditer(config.get('bz_regex'), changelog, re.IGNORECASE):
+                    idx = int(b.group(1))
```

Comment:
Is there a risk for b.group(1) to be None in case nothing is found ? If so that would make the casting break here

Reply:
If the regex doesn't match, we will never enter the for loop. If it matches, then `b.group(1)` cannot be None.

Reply:
Cool just wanted to make sure of that :+1: